checklists/alz_checklist.en.json

```diff
@@ -1220,127 +1183,309 @@
+            "text": "Optimise Rule Configuration and processing",
+            "waf": "Performance",
+            "service": "Firewall",
+            "guid": "828cec2e-af6c-40c2-8fa2-1b681ee63eb7",
+            "id": "D07.12",
+            "severity": "Medium",
+            "link": "https://learn.microsoft.com/en-us/azure/firewall-manager/rule-hierarchy"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Use IP Groups or IP prefixes to reduce number of IP table rules",
+            "waf": "Performance",
+            "service": "Firewall",
+            "guid": "0da83bb1-2f39-49af-b5c9-835fc455e3d1",
+            "id": "D07.12",
+            "severity": "Medium",
+            "link": "https://learn.microsoft.com/en-us/azure/firewall/ip-groups"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Avoid wildcards as sources for DNATS",
+            "waf": "Performance",
+            "service": "Firewall",
+            "guid": "c44c6f0e-1642-4a61-a17b-0922f835c93a",
+            "id": "D07.13",
+            "severity": "Medium",
+            "link": "https://learn.microsoft.com/en-us/azure/firewall/tutorial-firewall-dnat"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Prevent SNAT Port exhaustion",
+            "waf": "Performance",
+            "service": "Firewall",
+            "guid": "7371dc21-251a-47a3-af14-6e01b9da4757",
+            "id": "D07.14",
+            "severity": "Medium",
+            "link": "https://learn.microsoft.com/en-us/azure/nat-gateway/tutorial-hub-spoke-nat-firewall"
+        },
+         {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Enable TLS Inspection",
+            "waf": "Performance",
+            "service": "Firewall",
+            "guid": "346840b8-1064-496e-8396-4b1340172d52",
+            "id": "D07.15",
+            "severity": "High",
+            "link": "https://learn.microsoft.com/en-us/azure/firewall/premium-features#tls-inspection"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Consider Web Categories to allow or deny outbound access in bulk.",
+            "waf": "Performance",
+            "service": "Firewall",
+            "guid": "39990a13-915c-45f9-a2d3-562d7d6c4b7c",
+            "id": "D07.16",
+            "severity": "Low",
+            "link": "https://learn.microsoft.com/en-us/azure/firewall/premium-features#web-categories"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Use Azure Firewall in parallel/sequence with Azure App Gateway",
```

Comment:
This probably should transition to the app delivery checklist, as App Gateways are not part of the Platform Landing Zones being checked in this section.  I agree that it is a good recommendation, and if we feel like we need to make platform teams aware, could we change it to something like:

"As part of your TLS inspection, plan for receiving traffic from Azure App Gateways for inspection."

If we are giving guidance, then I think we should guide to use this: https://learn.microsoft.com/en-us/azure/architecture/example-scenario/gateway/application-gateway-before-azure-firewall

And then lets customers deviate from that at their own risk.